src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -2053,18 +2065,19 @@ def download_project(self, extensions=None, target_dir=None, progress_bar=False)
             List of downloaded files.
         """
         if not extensions:
-            files = self.list_files()
+            # files = self.list_files()
             list_of_files = self.download(
-                files, target_dir=target_dir, progress_bar=progress_bar
+                files="*", target_dir=target_dir, progress_bar=progress_bar
             )
```

Comment:
I'm not sure if I am happy with `*`. `mapdl.download` should download a file or list of files. But if nothing is specified... what should it happen? I guess raise an exception? Because we have already `download_project` for not having to specify files. Furthermore, `download_project` was intended for that use but avoiding to download big files such as `full`, `rst`, etc.

Reply:
Interesting! It could be removed then